### PR TITLE
Updates and additions to ISV Portal text fields

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
Updates and additions to match the new functionality coming in the ISV Portal.

- **Is this PR related to a Monolith release?**
2021.03.0

### Resolves:

* [OKTA-372050](https://oktainc.atlassian.net/browse/OKTA-372050)

### CODE REVIEW

**Review by eduardopedayo-okta — APPROVED** [review]
LGTM

**Review by tovamarkowitz-okta — APPROVED** [review]

**Review by azhariqbal-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md [hunk 1]
```diff
@@ -28,6 +28,8 @@
 
 * **Enter your Post Logout Redirect URI** &mdash; Provide the complete logout URI for your app integration, including the variable names if you're using a per tenant design.
 
-* **Optional: Link to configuration guide** &mdash; Your configuration guide (in either HTML or PDF format) should have step by step instructions on how to set up SSO between Okta and your systems. See [Prepare a customer-facing configuration guide](/docs/guides/submit-app/create-guide).
+* **To configure OIDC, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; If a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the set up time for your customers.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md:31]:
```suggestion
* **To configure OIDC, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; If a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the setup time for your customers.
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md:31]: setup as one word for adj and noun, as per our style guidelines

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md [hunk 2]
```diff
@@ -28,6 +28,8 @@
...
+
+* **Link to configuration guide** &mdash; Your configuration guide (in either HTML or PDF format) should have step by step instructions on how to set up SSO between Okta and your systems. See [Prepare a customer-facing configuration guide](/docs/guides/submit-app/create-guide).
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md:33]:
```suggestion
* **Link to configuration guide** &mdash; Your configuration guide (in either HTML or PDF format) should have step-by-step instructions on how to set up SSO between Okta and your systems. See [Prepare a customer-facing configuration guide](/docs/guides/submit-app/create-guide).
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md:33]: just suggestion, I like hyphens in step-by-step

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 * **Is SAML support available in the SP for all tenants by default or is it available only for specific SKUs?** &mdash; If you select **Certain SKUs**, provide details on which products provide SAML support.
 
-* **To configure SAML, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; If a customer needs support to configure your integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the set up time for your customers.
+* **To configure SAML, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; If a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the set up time for your customers.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md:7]:
```suggestion
* **To configure SAML, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; If a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the setup time for your customers.
```

**Review by brianduffield-okta — APPROVED** [review]
LGTM - just a suggestion and a couple style issues.

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md (hunk 2 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md:33]: Fixed here and elsewhere in this guide.

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md (hunk 1 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/openidconnect/submit.md:31]: Fixed here and in the other location in the guide where it is used.

File: packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md (hunk 3 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/saml2/submit.md:7]: Fixed

**Pull request merged**